sql/4_2_2-to-5_0_0_upgrade.sql

```diff
@@ -771,3 +771,13 @@ INSERT INTO `supported_external_dataloads` (`load_type`, `load_source`, `load_re
 ALTER TABLE `form_encounter` ADD `pos_code` tinyint(4) default NULL;
 #EndIf
 
+#IfNotColumnType drugs size varchar
+ALTER TABLE `drugs` CHANGE `size` `size` varchar(25) default NULL ;
+#EndIf
+
+#IfColumn prescriptions size
```

Comment:
Can you use IfNotColumnType here. maybe:
``````
#IfColumnType prescriptions size varchar(25)